rest/model/task_annotations.go

```diff
@@ -31,6 +35,10 @@ type APIIssueLink struct {
 	IssueKey *string    `bson:"issue_key,omitempty" json:"issue_key,omitempty"`
 	Source   *APISource `bson:"source,omitempty" json:"source,omitempty"`
 }
+type DisplayTicket struct {
```

Comment:
Should call this APIDisplayTicket. But also, why don't we add JiraTicket to APIIssueLink? Whenever we get the issue, won't we want to also retrieve ticket information?